server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -222,15 +222,15 @@ public ThreadPool(final Settings settings, MeterRegistry meterRegistry, final Ex
         builders.put(Names.ANALYZE, new FixedExecutorBuilder(settings, Names.ANALYZE, 1, 16, TaskTrackingConfig.DO_NOT_TRACK));
         builders.put(
             Names.SEARCH,
-            new FixedExecutorBuilder(settings, Names.SEARCH, searchOrGetThreadPoolSize, 1000, TaskTrackingConfig.DEFAULT)
+            new FixedExecutorBuilder(settings, Names.SEARCH, searchOrGetThreadPoolSize, 1000, new TaskTrackingConfig(true, 0.1))
```

Comment:
The default EWMA (exponential weighted moving average) alpha value is `0.3`. I've updated this to `0.1` be remain consistent with the `WRITE`,`SYSTEM_WRITE`, and `SYSTEM_CRITICAL_WRITE` threadpools which are used for index-load autoscaling.